docs/x509/reference.rst

```diff
@@ -2973,6 +3003,23 @@ instances. The following common OIDs are available as constants.
         :class:`~cryptography.x509.AccessDescription` objects.
 
 
+.. class:: SubjectInformationAccessOID
+
+    .. versionadded:: 3.0
+
+    .. attribute:: CA_REPOSITORY
+
+        Corresponds to the dotted string ``"1.3.6.1.5.5.7.48.5"``. Used as the
+        identifier for CA repository data in
+        :class:`~cryptography.x509.AccessDescription` objects.
+
+    .. attribute:: CA_ISSUERS
```

Comment:
This doesn't seem to exist?

Reply:
> This doesn't seem to exist?

Not as a valid `accessMethod` for SIA, anyways.

According to the syntax shown at the very end of section 4.2.2.2, the only valid OIDs for `accessMethod` (in SIA) are `1.3.6.1.5.5.7.48.5` ("`id-ad-caRepository`") and `1.3.6.1.5.5.7.48.3` ("`id-ad-timeStamping`"); the former is "used when the subject is a CA" and the latter is "used when the subject is an end entity".

`1.3.6.1.5.5.7.48.2` is "`id-ad-caIssuers`", though, so perhaps just a classic "off-by-one" error?  :smile:

---

On a (somewhat) related note, [section 3.1 of RFC4387](https://tools.ietf.org/html/rfc4387#section-3.1), identifies at least two other valid OIDs for `accessMethod` (in AIA/SIA):

* `1.3.6.1.5.5.7.48.6`: "`id-ad-http-certs`"
* `1.3.6.1.5.5.7.48.7`: "`id-ad-http-crls`"

Then again, RFC4387 predates RFC5280 by ~27 months yet `.6` and `.7` are absent from / not mentioned in RFC5280...

Reply:
RFC 5280 doesn't obsolete 4387, but it's still odd. I left this in accidentally though -- I'll just expose the one OID for now (we can wait another 5 years for someone to ask for these two).

Reply:
Right, RFC5280 doesn't mention or reference RFC4387 at all (leaving some unanswered questions, in my opinion).

I, myself, am quite happy with just "`id-ad-caRepository`". If I end up needing the `.48.6` or `.48.7` OIDs later, I'll send a PR then.

Thanks again!